File: java_projects/glue_code/commons-pool/src/test/java/org/apache/commons/pool2/PrivateException.java
package org.apache.commons.pool;

import org.graalvm.polyglot.Value;

public class PrivateException extends RuntimeException {
  private static final long serialVersionUID = 1L;
  private static Value clz =
      ContextInitializer.getPythonClass("/PrivateException.py", "PrivateException");
  private Value obj;

  public PrivateException(Value obj) {
    this.obj = obj;
  }

  public Value getPythonObject() {
    return obj;
  }

  public PrivateException(final String message) {
    //
    // super(message);
    //

    this.obj = clz.invokeMember("__init__", message);
  }
}

File: data/recomposed_projects/codellama-13b-instruct/commons-pool/src/test/org/apache/commons/pool2/PrivateException.py
# Imports Begin
import unittest

# Imports End


class PrivateException(unittest.TestCase, RuntimeException):

    # Class Fields Begin
    __serialVersionUID: int = 1
    # Class Fields End

    # Class Methods Begin
    def __init__(self, message: str) -> None:

        super().__init__(message)

    # Class Methods End
